<launch>  
  <param name="goal_x" value="18.5" />
  <param name="goal_y" value="27.5" />
  <param name="goal_theta" value="45" />

  <param name="/use_sim_time" value="true"/>
  <include file="$(find send_goals)/move_base_config/move_base.xml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(find send_goals)/stage_config/maps/willow-full-0.05.pgm 0.05"/>
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find send_goals)/stage_config/worlds/willow-pr2-5cm.world"/>   
  <include file="$(find send_goals)/move_base_config/amcl_node.xml"/> 

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find send_goals)/single_robot.rviz" /> 

  <node name="send_goals_node" pkg="send_goals" type="send_goals_node" output="screen"/>
</launch>

